tags/tags.toml

```
@@ -895,3 +895,12 @@ content = """
 + embed.setFooter({ text: 'This is an example text', iconURL: 'https://example.com/icon.png' })
 ```
 """
+
+[interaction-acknowledged]
+keywords = ["interaction-acknowledged", "interaction", "acknowledged", "reply", "deferReply"]
```

```suggestion
keywords = ["already-acknowledged", "interaction-acknowledged"]
```
keywords need to be unique, some of these were way too generic